contracts/FIFOMarket.sol

```diff
@@ -133,13 +151,113 @@ contract FIFOMarket is
   }
 
   /**
-   * @notice The amount of supply available for anyone to buy.
+   * @notice Gets the active restricted or unrestricted supply currently listed for sale in the market.
+   */
+  function activeSupply(bool includeRestricted)
+    external
+    view
+    returns (ActiveSupply memory)
+  {
+    address supplierAddress = _currentSupplierAddress;
+    ActiveSupplier[] memory suppliers = new ActiveSupplier[](
+      activeSupplierCount
+    );
+    address[] memory fifoMarketAddressArray = new address[](
```

Comment:
why still using fifo in terminology?

Reply:
It's all semantics at this point, but technically suppliers are still ordered in the queue by FIFO.